File: java/com/baidu/ai/aip/face/Delete.java
/*
 * Copyright (C) 2017 Baidu, Inc. All Rights Reserved.
 */
package com.baidu.ai.aip.face;

import com.baidu.ai.aip.utils.HttpUtil;

/**
 * 人脸查找——删除
 */
public class Delete {

    public static void main(String[] args) {
        // 人脸查找——删除 url
        String deleteUrl = "https://aip.baidubce.com/rest/2.0/faceverify/v1/faceset/user/delete";
        try {
            // 请求参数
            // 用户ID
            String uid = "test_user_5";
            String params = "uid=" + uid;
            /**
             * 线上环境access_token有过期时间， 客户端可自行缓存，过期后重新获取。
             */
            String accessToken = "#####调用鉴权接口获取的token#####";
            String result = HttpUtil.post(deleteUrl, accessToken, params);
            System.out.println(result);
        } catch (Exception e) {
            e.printStackTrace();
        }
    }
}

File: python/face/delete.py
# encoding:utf-8
import base64
import urllib
import urllib2

from token import access_token

'''
人脸查找之删除接口
'''

deleteUrl = "https://aip.baidubce.com/rest/2.0/faceverify/v1/faceset/user/delete"
# 参数uid：用户id（由数字、字母、下划线组成），长度限制128B
params = {"uid": "testuid"}
params = urllib.urlencode(params)
access_token = access_token.AuthService()
deleteUrl = deleteUrl + "?access_token=" + access_token
request = urllib2.Request(url=deleteUrl, data=params)
request.add_header('Content-Type', 'application/x-www-form-urlencoded')
response = urllib2.urlopen(request)
content = response.read()
if content:
    print content
